client/blocks/eligibility-warnings/datacenter-picker.tsx

```diff
@@ -0,0 +1,155 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import styled from '@emotion/styled';
+import { Button } from '@wordpress/components';
+import { localize, LocalizeProps, translate } from 'i18n-calypso';
+import { useState } from 'react';
+import ExternalLink from 'calypso/components/external-link';
+import FormRadiosBar from 'calypso/components/forms/form-radios-bar';
+
+interface ExternalProps {
+	value: string;
+	onChange: ( newValue: string ) => void;
+}
+
+type Props = ExternalProps & LocalizeProps;
+
+const DatacenterOptions = [
+	{
+		value: 'ams',
+		label: translate( 'Amsterdam' ),
+		thumbnail: {
+			imageUrl: '',
+		},
+	},
+	{
+		value: 'bur',
+		label: translate( 'California' ),
+		thumbnail: {
+			imageUrl: '',
+		},
+	},
+	{
+		value: 'dfw',
+		label: translate( 'Texas' ),
+		thumbnail: {
+			imageUrl: '',
+		},
+	},
+	{
+		value: 'dca',
+		label: translate( 'Washington, D.C.' ),
+		thumbnail: {
+			imageUrl: '',
+		},
+	},
+];
+
+const Form = styled.form( {
+	maxWidth: '564px',
+} );
+
+const FormHeadingContainer = styled.div( {
+	display: 'flex',
+	alignItems: 'center',
+	marginBottom: '8px',
+} );
+
+const FormHeading = styled.h3( {
+	fontWeight: 600,
+	marginRight: '8px',
+} );
+
+const FormDescription = styled.p( {
+	marginBottom: '12px',
+} );
+
+const AutomaticFormLabel = styled.label( {
+	display: 'block',
+	marginBottom: '10px',
+	width: '100%',
+	maxWidth: '541px',
```

Comment:
This max width works for the usual layout, but not when the screen is narrower. Nothing breaks though, and you probably don't want to come up with multiple SVGs for all the different layouts.

Reply:
You should also add `cursor: pointer` to `<AutomaticFormLabel>` so that it matches `<FormRadiosBar>`

Reply:
> You should also add `cursor: pointer` to `<AutomaticFormLabel>` so that it matches `<FormRadiosBar>`

ff53dc2e577e2c7496c7cddf4f8016e480728d05